src/autocomplete.js

```diff
@@ -206,7 +207,7 @@ export default class Autocomplete extends Component {
 
     this.setState({ query })
 
-    const searchForOptions = !queryEmpty && queryChanged && queryLongEnough
+    const searchForOptions = showAllValues || (!queryEmpty && queryChanged && queryLongEnough)
```

Comment:
Yeah I can see the logic however I think it's not intuitive to just have minlength 0 trigger quite different behaviour to the other min lengths. Especially if we were to look into styling it slightly differently which I think is needed